tests/device.c

```diff
@@ -51,7 +51,7 @@ void device1 (NN_UNUSED void *arg)
 
     /*  Run the device. */
     rc = nn_device (deva, devb);
-    nn_assert (rc < 0 && (nn_errno () == ETERM || nn_errno () == EBADF));
+    nn_assert (rc < 0 && (nn_errno () == ETERM));
```

Comment:
Good to see the test tighten up here! If the previous pass criteria would switch between the two error codes, yet now it's just one, that seems like a step in a good direction!